server/src/types/types.py

```diff
@@ -22,6 +22,12 @@ def insert(data: dict) -> None:
             raise ValueError(f'Cannot insert data of type{type(data)}')
         query.insert('users', data)
 
+    def find_all(filters={}) -> list:
+        return query.find_all('users', filters)
+
```

Comment:
why do we need this here lol? the one in query has filters={} by default as well so we could use that